README.md

```diff
@@ -105,89 +22,38 @@ In the examples below the audit record is prepended with a timestamp which is cr
-
- * [Throughput](https://rawgit.com/Ericsson/ecaudit/release/c3.0/doc/ecaudit-performance.html)
- * [Latency](https://rawgit.com/Ericsson/ecaudit/release/c3.0/doc/ecaudit-performance.html?stats=undefined&metric=mean&operation=WRITE&smoothing=1&show_aggregates=true&xmin=0&xmax=91.08&ymin=0&ymax=0.33)
+Checkout the detailed [description](doc/description.md) for a more comprehensive list of features, limitations and operational impact.
 
-Refer to the guides of LOGBack settings, authentication caches and whitelist settings to get best possible performance.
 
+## Getting Started
 
-## Connections
+ecAudit integrates with Apache Cassandra using its existing plug-in points.
 
-A typical Cassandra client will connect to many (if not all) nodes in the Cassandra cluster.
-For this reason audit records will be created on several places when a new client connect to a cluster.
-Further, it is not uncommon for a client to set up several connections to a server which will result in several audit records for user authentication.
 
-Some clients will also make a few internal queries to the internal system tables in Cassandra.
-These queries will appear as any other audit record even though they were not generated by the actual application logic at the client.
+### Download
 
+Official releases of ecAudit can be downloaded from Maven Central.
+Get the ecAudit flavor for your Cassandra version.
 
-## Known Limitations
+[![ecAudit for Cassandra 3.11.<latest>](https://img.shields.io/maven-central/v/com.ericsson.bss.cassandra.ecaudit/ecaudit_c3.11.svg?label=ecAudit%20for%20Cassandra%203.11.<latest>)](https://search.maven.org/search?q=g:%22com.ericsson.bss.cassandra.ecaudit%22%20AND%20a:%22ecaudit_c3.11%22)
+[![ecAudit for Cassandra 3.0.<latest>](https://img.shields.io/maven-central/v/com.ericsson.bss.cassandra.ecaudit/ecaudit_c3.0.svg?label=ecAudit%20for%20Cassandra%203.0.<latest>)](https://search.maven.org/search?q=g:%22com.ericsson.bss.cassandra.ecaudit%22%20AND%20a:%22ecaudit_c3.0%22)
+[![ecAudit for Cassandra 3.0.11](https://img.shields.io/maven-central/v/com.ericsson.bss.cassandra.ecaudit/ecaudit_c3.0.11.svg?label=ecAudit%20for%20Cassandra%203.0.11)](https://search.maven.org/search?q=g:%22com.ericsson.bss.cassandra.ecaudit%22%20AND%20a:%22ecaudit_c3.0.11%22)
 
+For a detailed description of compatible Cassandra versions, refer to the [Cassandra Compatibility Matrix](doc/cassandra_compatibility.md).
 
-### Interfaces
 
-ecAudit does not provide auditing on the JMX interface.
-ecAudit does not provide auditing functionality for prepared statements on the legacy RPC (Thrift) interface in Cassandra.
-However, regular (un-prepared) statements on the RPC (Thrift) interface are audit logged.
+### Setup
 
+Install and configure ecAudit using the setup guide for your Cassandra version.
 
-### The USE statement
+* [ecAudit Setup Guide for Cassandra 3.11.<latest>](https://github.com/Ericsson/ecaudit/blob/release/c3.11/doc/setup.md)
```

Comment:
this link does not work, but I guess it will start working once the code is merged there (to that branch) ?

Reply:
Actually a typo. Fixed!